src/style.css

```diff
@@ -1,3 +1,74 @@
+:root {
+  --blue-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
+  text-transform: uppercase;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: auto;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  height: 60px;
+}
+
+.nav_item {
+  position: relative;
+  margin: 0 20px 0 0;
```

Comment:
You are using `display:flex` for parent container. 
This makes usage of margin-right redundant, you can(and should) just use `gap` for parent. 

As a rule of thumb: wherever possible, parent should be responsible for positioning it's children within itself, NOT children with margins || paddings || etc.